keanu-python/tests/test_base.py

```diff
@@ -0,0 +1,80 @@
+import keanu as kn
+import numpy as np
+import pytest
+from py4j.java_gateway import java_import
+
+jvm_view = kn.KeanuContext().jvm_view()
+java_import(jvm_view, "io.improbable.keanu.vertices.dbl.probabilistic.GaussianVertex")
+
+
+def test_can_pass_scalar_to_vertex():
+    gaussian = kn.Vertex(jvm_view.GaussianVertex, (0., 1.))
+    sample = gaussian.sample()
+
+    assert sample.isScalar()
+
+
+def test_can_pass_np_tensor_to_vertex():
+    gaussian = kn.Vertex(jvm_view.GaussianVertex, (np.array([[0.1, 0.4]]), np.array([[0.4, 0.5]])))
+    sample = gaussian.sample()
+
+    shape = sample.getShape()
+    assert sample.getRank() == 2
+    assert shape[0] == 1
+    assert shape[1] == 2
+
+
+def test_can_pass_java_object_to_vertex():
```

Comment:
but you can't pass any old java object, can you? I think this should be called `test_can_pass_another_vertex_to_the_vertex `

Reply:
You can. And if the constructor signature doesn't take that specific java object, it'll throw an error on the java side.